FEATURE_GATES.md

```diff
@@ -61,7 +61,7 @@ Features that reach GA and over time become stable will be removed from this tab
 | CombinedRoutes   | `true`  | Beta  | 2.8.0   | TBD   |
 | GatewayAlpha     | `false` | Alpha | 2.6.0   | TBD   |
 | ExpressionRoutes | `false` | Alpha | 2.10.0  | TBD   |
-| CombinedServices | `false` | Alpha | 2.10.0  | TBD   |
+| CombinedServices | `true`  | Beta  | 2.10.0  | TBD   |
```

Comment:
Should be `2.11.0` here?

Reply:
This column is `Since` version, so it should stay as it is.

Reply:
Yeah I wasn't sure about this, and it actually looks inconsistent, and some entries list initial addition whereas others list current phase.

Neither looks correct looking at the Kubernetes docs, which list each stage. I added 5e9c1b6b458e751ff03a039b4010ea97cbd56af6 to backfill those.